imod/mf6/wel.py

```diff
@@ -207,7 +207,7 @@ def __init__(
         repeat_stress: Optional[xr.DataArray] = None,
     ):
         if id is None:
-            id = np.arange(len(x)).astype(str)
+            id = list(np.arange(len(x)).astype(str))
```

Comment:
Nitpick, in this case I'd suggest:
`ids = [str(i) for i in range(len(x))]`

`id` is also a built-in function of Python that you shouldn't redefine.

Reply:
done, now using `[str(i) for i in range(len(x))]`. 
`id` is the name of a constructor argument, so I'm hesitant to rename it as it may break user scripts.

Reply:
That's indeed a good argument to keep `id` (it's not like we especially need the `id` in this function's scope).